lib/orderbook/OrderBook.ts

```diff
@@ -257,10 +258,14 @@ class OrderBook extends EventEmitter {
     }
   }
 
-  private isOwnOrder = (type: OrdersMap) => {
+  private isOwnOrdersMap = (type: OrdersMap) => {
     return type === this.ownOrders;
   }
 
+  private isOwnOrder = (order: orders.StampedOrder) => {
+    return !order['hostId'];
```

Comment:
Daniel will remove the property `hostId` in #256 so we should probably use `peerId` to distinguish `peerOrder` and `ownOrder`.

Reply:
True, although I guess it should be handled in whichever gets merged second.